docs/sources/next/results-output/web-dashboard/_index.md

```diff
@@ -65,6 +65,8 @@ To automatically generate a report from the command line once the test finishes
 K6_WEB_DASHBOARD=true K6_WEB_DASHBOARD_EXPORT=html-report.html k6 run script.js
 ```
 
+The report only includes graphs if the test duration is over thirty seconds.
```

Comment:
@heitortsergent it would be a bit more precise if write something like this instead of 30 sec: "3 times the aggregation period (default 10sec). "
This is how 30 sec come into the picture